functions/zoom-downloader.py

```diff
@@ -38,23 +60,43 @@ def handler(event, context):
     logger.info(event)
 
     download_queue = sqs.get_queue_by_name(QueueName=DOWNLOAD_QUEUE_NAME)
-    logger.debug("got queue {}".format(str(download_queue)))
 
     try:
-        logger.debug("fetching a message...")
-        messages = download_queue.receive_messages(MaxNumberOfMessages=1,
-                                                   VisibilityTimeout=700)
-        logger.debug("Messages received from download queue: {}".format(messages))
-        message = messages[0]
-        logger.debug({'queue_message': message})
+        messages = download_queue.receive_messages(MaxNumberOfMessages=1)
+        download_message = messages[0]
+        logger.debug({'queue message': download_message})
 
     except IndexError:
-        logger.warning("No uploads ready for processing")
+        logger.info("No recordings ready for processing")
```

Comment:
It would probably be more accurate to say "No download queue messages received" since the queue could be blocked by an invisible message.